CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.12.0
+
+## Added
+
+- [[#187](https://github.com/rust-vmm/kvm-ioctls/pull/187)] Support for 
+  `KVM_SET_IDENTITY_MAP_ADDR`
+- Derive Debug for all exported structs and enums
+- [[#189](https://github.com/rust-vmm/kvm-ioctls/pull/189)] Expose `KVM_SET_` and 
+  `KVM_HAS_DEVICE_ATTR` for vcpu
+- [[#191](https://github.com/rust-vmm/kvm-ioctls/pull/191)] Add `KVM_TRANSLATE` and
+  `translate_gva` support
+- [[#190](https://github.com/rust-vmm/kvm-ioctls/pull/190)] Enable usage of `sync_regs`
```

Comment:
I would add a bit more text here: "... that allow synchronizing registers without the need to use an ioctl".

Reply:
Done!